kv/memberlist/memberlist_client.go

```diff
@@ -956,29 +966,71 @@ func (m *KV) NotifyMsg(msg []byte) {
 		return
 	}
 
-	// we have a ring update! Let's merge it with our version of the ring for given key
-	mod, version, err := m.mergeBytesValueForKey(kvPair.Key, kvPair.Value, codec)
+	ch := m.getKeyWorkerChannel(kvPair.Key)
+	select {
+	case ch <- valueUpdate{
+		value:       kvPair.Value,
+		codec:       codec,
+		messageSize: len(msg),
+	}:
```

Comment:
Nit: It took me a second to understand the final `:` here :) Perhaps we can format this on a single line?

Reply:
fair point, changed @ 7e18983e49f09e4e290e9c7640419a3c32839915